operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -5774,6 +5774,9 @@ rules:
   - apiGroups: ["authorization.k8s.io"]
     resources: ["subjectaccessreviews"]
     verbs: ["create"]
+  - apiGroups: ["multicluster.x-k8s.io"]
```

Comment:
Not sure if it is needed for this PR.

Reply:
True, it is not needed, I can remove. The `make gen` generated this. Shall I delete it?

Reply:
Please rebase with master and then run `make gen`. It's not needed so you can remove it.